api/getProfile/index.test.ts

```diff
@@ -1,14 +1,51 @@
-const httpFunction = require('./index');
-const context = require('../testing/defaultContext')
+/* eslint-disable no-unused-vars */
+import httpTrigger from '.'
+import { UserInfo } from '../models/UserInfo'
+import { Context, HttpRequest, Logger } from '@azure/functions'
+/* eslint-enable no-unused-vars */
 
 test('Http trigger should return known text', async () => {
+  const request = {
+    method: 'GET',
+    url: '',
+    headers: {},
+    query: {},
+    params: { userId: 'pete' }
+  } as HttpRequest
 
-    const request = {
-        params: { userId: 'bill' }
-    };
+  const user = {
+    userId: 'pete',
+    aboutMe: 'about',
+    interests: 'interests',
+    skills: 'skills'
+  } as UserInfo
 
-    await httpFunction(context, request);
+  const context = {
```

Comment:
this seems excessive, is there not a context object/class we can extend/instantiate from?

Reply:
context is just an interface and there are no implementations exported as part of the @azure/functions package. We could create a class that implements the interface to abstract this, which is what they are doing in the test for the package. I was honestly being lazy on this PR since I wasn't using it anywhere else yet, but I can create a class as part of this PR if you think it makes sense to do so now.

Reply:
Nope this is fine for now! 😄